lib/screens/route_patterns/route_pattern.ex

```diff
@@ -76,6 +76,7 @@ defmodule Screens.RoutePatterns.RoutePattern do
     params = %{
       "include" => "representative_trip.stops,route",
       "filter[stop]" => stop_id,
+      "filter[direction_id]" => 0,
```

Comment:
Is it true that we will always want the stop sequence in the same direction?

Reply:
Yes, ha. 

We only need an ordered sequence of stops. Doesn't matter which way they are, EB or WB. In the case of the GL, they all need to go in the same direction as each other. So setting 0 or 1 would fix this